synapse/cortex.py

```diff
@@ -1007,7 +1008,24 @@ async def getSpawnInfo(self):
                 'pkgs': await self.getStormPkgs(),
             },
             'model': await self.getModelDefs(),
+            'spawncorector': await self.getSpawncoreCtor()
         }
+        addl = await self._addlSpawnInfo()
+        ret.update(**addl)
+        return ret
+
+    async def _addlSpawnInfo(self):
+        '''
+        Additional spawninfo data.
+        '''
+        return {}
+
+    async def getSpawncoreCtor(self):
+        if self.spawncorector:
+            return self.spawncorector
+        import synapse.lib.spawn as s_spawn
+        self.spawncorector = f'{s_spawn.__name__}.{s_spawn.SpawnCore.__name__}'
```

Comment:
This is no less hard-coded than simply returning "synapse.lib.spawn.SpawnCore".  Also i'm not a fan of having an accessor method *and* a local value. :)